### PR TITLE
create process for each rule, allow to define wait for each rule

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #74

This PR allow to create processes for each errors found.

### CODE REVIEW

File: test/controllers/event_controller_test.exs [hunk 1]
```diff
@@ -21,4 +22,10 @@ defmodule Dwylbot.EventTestController do
       assert json_response(conn, 200)
     end
   end
+
+  test "helper wait", _conn do
+      assert Helpers.wait(:prod, 1, 2, 3) == 1
+      assert Helpers.wait(:dev, 1, 2, 3) == 2
+      assert Helpers.wait(:test, 1, 2, 3) == 3
+  end
```
Comment (samhstn) [inline test/controllers/event_controller_test.exs]: You already have a `@doc` test for this function, you shouldn't have both. I'd say get rid of this one and have

```elixir
doctest Dwylbot.Rules.Helpers
```

At the top of this file

File: web/controllers/rules/helpers.ex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  wait: define the time for our rules to wait depeding on the environment
+  """
+  @doc """
+    iex>waith(:prod, 1, 2, 3)
+    1
+    iex>waith(:dev, 1, 2, 3)
+    2
+    iex>waith(:test, 1, 2, 3)
+    3
+  """
```
Comment (samhstn) [inline web/controllers/rules/helpers.ex]: Nice, I think you've got a typo `waith -> wait`. You should also run this doctest in the test file for this

**Review by samhstn — COMMENTED** [review]
Will looks great once those `doc`tests are running

File: test/controllers/event_controller_test.exs (hunk 1 above)
Comment (SimonLab) [reply test/controllers/event_controller_test.exs]: Thanks @shouston3 I knew I was missing something to run the doctest properly

**Review by samhstn — APPROVED** [review]
Looks great 👍

**Review by nelsonic — APPROVED** [review]
@SimonLab great work! 🎉

**Pull request merged**